lib/Analysis/DependenceAnalysis.cpp

```diff
@@ -131,17 +131,35 @@ circt::analysis::MemoryDependenceAnalysis::MemoryDependenceAnalysis(
   // Collect affine loops grouped by nesting depth.
   std::vector<SmallVector<AffineForOp, 2>> depthToLoops;
   mlir::affine::gatherLoops(funcOp, depthToLoops);
+  depthToLoops.erase(depthToLoops.begin());
 
   // Collect load and store operations to check.
```

Comment:
Given the other discussion, can you please update this comment to say something like "Collect load and store operations in the root of the FuncOp".